lambdas/models/staging_metadata.py

```diff
@@ -19,6 +19,7 @@ class MetadataFile(BaseModel):
     )
 
     file_path: str = Field(alias="FILEPATH")
+    stored_file_name: Optional[str] = Field(alias="STORED-FILE-NAME", default=None)
```

Comment:
As stored_file_name should always have a value, it should not be Optional, nor have a default.
```suggestion
    stored_file_name: str = Field(alias="STORED-FILE-NAME")
```